website/TRANSLATIONS.md

```diff
@@ -0,0 +1,39 @@
+# Contributing Guidelines for our docs
+
+Are you a native speaker of a language other than English? We'd love to have your help!
+
+## How to contribute
+
+(Read the [contribution guidelines](../CONTRIBUTING.md) first for general information about contributing to this project.)
+
+### Initial Translation
+
+If the docs are not available in your language yet and you would like to translate them, you can do so by following these steps:
+
+1. Copy the `src/content/docs` directory to a new directory (exclude all other translated code, e.g. `ja`, `pt-br`, etc.) with the name of your language, e.g. `src/content/docs/de` for German. (If you're not sure what the language code is, you can find it [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).)
+
+2. File a draft PR to let others know you're working on it. This is to avoid duplicate work, and also allows for community feedback along the way.
+
+3. File by file, translate the content of the files.
+
+4. Update the `lang` attribute in the frontmatter of the files translated to your language. If there are any files you haven't translated yet, that's fine. Just leave the `lang: en` attribute in the frontmatter.
+
+5. If the language you are translating into is read right-to-left (for example, Arabic or Hebrew), also add `dir: rtl` to the frontmatter.
```

Comment:
I think with starlight the configuration is different

Reply:
Yup, the create-t3-app web was created before Starlight was born into this fantastic world. I need to adapt this part as well. I'm grateful for GitHub having a review feature.